app/models/job/Job.scala

```diff
@@ -217,10 +233,11 @@ class JobDAO @Inject()(sqlClient: SQLClient)(implicit ec: ExecutionContext)
     } yield ()
   }
 
-  def countByStatus: Fox[Map[String, Int]] =
+  def countByState: Fox[Map[String, Int]] =
     for {
       result <- run(sql"""select state, count(_id)
                            from webknossos.jobs_
+                           where manualState is null
                            group by state
                            order by state
                            """.as[(String, Int)])
```

Comment:
Not sure what this is used for, but shouldn't we also count the manualState fields and add those, since they have precedence?